src/libsync/CMakeLists.txt

```diff
@@ -69,6 +71,13 @@ if (WIN32)
     )
     add_definitions(-D_WIN32_WINNT=_WIN32_WINNT_WIN10)
     list(APPEND OS_SPECIFIC_LINK_LIBRARIES cldapi)
+elseif(LINUX) # elseif(LINUX OR APPLE)
```

Comment:
@er-vin Are you sure this is a proper define for Linux and Mac?

In QT5KeyChainConfig.make they use `if(UNIX AND NOT APPLE)` to distinguish between Unix(Linux) and Mac.

Reply:
Yes, I'm pretty sure. This is coming from our cmake/modules/DefinePlatformDefaults.cmake. Using UNIX you would catch also all the BSD variants and such which we really don't want since they don't have the same xattr API.